### ISSUE TITLE
`Deriv_(st[[2]], x, env, use.D)` in result of Deriv

### ISSUE THREAD

**Issue by notEvil** [sgsokol/Deriv#1]
Please take a look at the following code. It seems there is something going wrong during simplification. My guess for the source of issue are the superfluous brackets enclosing pnorm and dnorm.

```
f = parse(text='(1 + ((pnorm(y1, beta1 + beta2 * x + beta3 * x^2, 1))^(-alpha) - 
    1 + (pnorm(y2, beta4 * x + beta5 * x^3 + beta6 * x^4, 1))^(-alpha) - 
    1))^(((-1/alpha) - 1) - 1) * (((-1/alpha) - 1) * ((pnorm(y2, 
    beta4 * x + beta5 * x^3 + beta6 * x^4, 1))^((-alpha) - 1) * 
    (-alpha))) * ((-1/alpha) * ((pnorm(y1, beta1 + beta2 * x + 
    beta3 * x^2, 1))^((-alpha) - 1) * (-alpha))) * (1 * (dnorm(y1, 
    beta1 + beta2 * x + beta3 * x^2, 1)) * (dnorm(y2, beta4 * 
    x + beta5 * x^3 + beta6 * x^4, 1)))')[[1]]
env = list(y1=0, y2=0, beta1=1, beta2=1, beta3=1, beta4=1, beta5=1, beta6=1, x=0, alpha=0.5)
eval(f, env)

d = Deriv::Deriv(f, 'beta1')
eval(d, env)
d
```

**Comment by notEvil** [comment]
Of course, I found what the Deriv_... should be. `pnorm` is not in the drule environment. Why this doesn't stop Deriv from returning something I can't tell. Usually it complains as soon as there is some rule missing.

for now I use the following rule which seems valid (checked with numDeriv). You should consider adding this to the builtins

```
assign('pnorm', alist(q=dnorm(q, mean, sd), mean=-dnorm(q, mean, sd), sd=dnorm(q, mean, sd)*(mean - q)/sd), envir=Deriv::drule)
```

btw, great work

**Comment by sgsokol** [comment]
It was Simplify() who catched unduly an error message about unknown function.
It is fixed in the version 3.5
Thx
